### DISCUSSION TITLE
Configuring Weblate to Transmit Only Translated Strings

### DISCUSSION THREAD

**Discussion by LifeIsAParadox** [WeblateOrg/weblate#9258] [Q&A] [answered]
I have a problem where untranslated strings or empty strings are being pushed. Is there a way to configure Weblate to only transmit strings that have been translated?

**Comment by nijel** [comment, marked as answer]
Generally no (see https://github.com/WeblateOrg/weblate/issues/3745), but for monolingual translations, empty strings can be omitted from translation files using https://docs.weblate.org/en/latest/admin/addons.html#remove-blank-strings

**Reply by LifeIsAParadox** [reply]
**[Remove blank strings](https://docs.weblate.org/en/latest/admin/addons.html#remove-blank-strings)** addon works, thanks
